apps/src/lib/util/AnalyticsConstants.js

```diff
@@ -86,7 +86,9 @@ const EVENTS = {
   TA_RUBRIC_ON_STUDENT_WORK_UNLOADED: 'TA Rubric On Student Work Unloaded',
   TA_RUBRIC_SUBMITTED: 'TA Rubric Submitted',
 
+  // Hour of Code
   AGE_21_SELECTED_EVENT: 'Age 21+ Selected',
+  GUIDE_SENT_EVENT: 'Guide Sent',
```

Comment:
I searched Amplitude for an event with this name but didn't find one, do we have one set up for this?

Reply:
I can add it now.

Reply:
Done!

Reply:
Thanks Dani!